origin-js/scripts/serve.js

```diff
@@ -17,4 +19,141 @@ const start = async () => {
   await startIpfs()
 }
 
+
+class StoreMock {
+  constructor() {
+    this.storage = {}
+  }
+
+  get(key) {
+    return this.storage[key]
+  }
+
+  set(key, value) {
+    this.storage[key] = value
+  }
+}
+
+const createSamples = async() => {
+  const provider = new Web3.providers.HttpProvider('http://localhost:8545')
+  const web3 = new Web3(provider)
+  const accounts = await web3.eth.getAccounts()
+  const contractService = await contractServiceHelper(web3)
+  const ipfsService = new IpfsService({
+    ipfsDomain: '127.0.0.1',
+    ipfsApiPort: '5002',
+    ipfsGatewayPort: '8080',
+    ipfsGatewayProtocol: 'http'
+  })
+  const store = new StoreMock()
+  store.set(
+    'notification_subscription_start',
+    new Date().getTime()
+  )
+  marketplace = new Marketplace({
+    contractService,
+    ipfsService,
+    affiliate: accounts[3],
+    arbitrator: accounts[4],
+    store,
+    blockEpoch: await web3.eth.getBlockNumber()
+  })
+
+  //hawai-house
+  await marketplace.createListing({
+    listingType: 'unit',
```

Comment:
As opposed to hardcoding the listing data, how about using the [sample fixtures](https://github.com/OriginProtocol/origin/tree/master/origin-js/test/fixtures) ?